dv-export/python/query-changes/changes_fetcher.py

```diff
@@ -0,0 +1,124 @@
+import copy
+import logging
+import math
+from datetime import datetime
+from typing import Any
+
+from visier.api import ModelApiClient, QueryApiClient
+from visier.connector import VisierSession
+
+from constants import *
+
+logger = logging.getLogger(__name__)
+
+
+class ChangesFetcher:
+    """Fetches changes for full time existence of subject."""
+
+    DEFAULT_QUERY_LIMIT = 1000
```

Comment:
I think this might be too low. Should probably be at least around 10k, maybe 100k

Reply:
Update to 100K